doxygen/rpc.dox

```diff
@@ -87,7 +87,8 @@
  *
  * We can generate the C++ code by running:
  *
- *     protoc/ola_protoc --cppservice_out ./ Dictionary.proto
+ *     protoc --plugin=protoc-gen-cppservice=protoc/ola_protoc_plugin" \
```

Comment:
Unmatched quote. Also what's breaking the windows build is the lack of EXEEXT, but I guess that's taken for granted in a code example?